include/emp/Evolve/SystematicsAnalysis.hpp

```diff
@@ -19,24 +19,27 @@
 
 namespace emp {
 
+  /// @returns the taxon with the highest fitness out of any active taxon
+  /// in the given systematics manager.
+  /// @param s the systematics manager to search in. Must have more than 0 active taxa.
   template<typename systematics_t>
   Ptr<typename systematics_t::taxon_t> FindDominant(systematics_t & s) {
-    double best = -999999;
```

Comment:
Are these some diffs that leaked in from other work that the doc changes were cherry picked from?